src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_loop_emitters.cpp

```diff
@@ -151,34 +181,54 @@ void jit_loop_end_emitter::emit_impl(const std::vector<size_t>& in,
     std::copy(in.begin(), in.end() - 1, std::back_inserter(data_ptr_reg_idxs));
 
     auto reg_work_amount = XReg(in.back());
-    if (!evaluate_once) {
+    XReg reg_runtime_params = XReg(Operand::X0);
+    XReg reg_aux = h->X_TMP_1;
+
+    auto apply_increments = [&](const std::vector<int64_t>& increments_vec,
+                                const std::vector<bool>& incremented_vec,
+                                const std::vector<int64_t>& sizes_vec,
```

Comment:
By the way, since lamba function has capture clause with `&`, we can remove unchangeable parameters of function: `incremented_vec` and `sizes_vec`. They are always equal to `is_incremented` and `data_sizes`